dashboard/lib/services/lti.rb

```diff
@@ -97,7 +97,7 @@ def self.parse_nrps_response(nrps_response)
     members = nrps_response[:members]
     context_title = nrps_response.dig(:context, :title)
     members.each do |member|
-      next if member[:status] == 'Inactive' || member[:roles].exclude?(Policies::Lti::CONTEXT_LEARNER_ROLE)
+      next if member[:status] == 'Inactive'
```

Comment:
Does this result in the teacher adding themselves as their own follower, or is there logic somewhere to prevent that?